libgalois/src/DynamicBitset.cpp

```diff
@@ -108,6 +108,21 @@ katana::DynamicBitset::count() const {
   return ret.reduce();
 }
 
+size_t
+katana::DynamicBitset::SerialCount() const {
+  size_t ret = 0;
+  for (uint64_t n : bitvec_) {
+#ifdef __GNUC__
```

Comment:
Will it take this path in `clang` too?

Reply:
I'm not actually sure; I just copied what was there in the other version.